packages/components/src/2016-12-01/datepicker.js

```diff
@@ -108,6 +108,8 @@ class Datepicker extends Element {
   _handleKeydown(event) {
     if (eventKey(event) === 'Tab') {
       this._dispatchStateChange('leaveFocused');
+    } else {
```

Comment:
I spent some time trying to test this - since our standard is to have test for bugs. However there is no good way to simulate a keypress event that would actually write the value to the input field, that does not feel super removed from what the actual browser behavior would look like.